include/libopencm3/stm32/f4/rcc.h

```diff
@@ -1095,6 +1101,7 @@ void rcc_set_main_pll_hsi(uint32_t pllm, uint32_t plln, uint32_t pllp,
 void rcc_set_main_pll_hse(uint32_t pllm, uint32_t plln, uint32_t pllp,
 			  uint32_t pllq, uint32_t pllr);
 uint32_t rcc_system_clock_source(void);
+void rcc_clock_setup_pll(const struct rcc_clock_scale *clock);
 void rcc_clock_setup_hse_3v3(const struct rcc_clock_scale *clock);
```

Comment:
you can add an attribute deprecated here if you like, see f3 rcc.c, or you can add @deprecated doxygen to the comments to mark it in the docs.  (you can even do both!) :)

Reply:
Added the `@deprecated` coment: 65a00fe49ec5f65f9a838d03d8a0ba1b12ff8cec

Reply:
Resolving this conversation, if you don't mind.